bokeh/core/property/color.py

```diff
@@ -66,14 +66,12 @@ def validate(self, value, detail=True):
 class Color(Either):
     """ Accept color values in a variety of ways.
 
-    For colors, because we support named colors and hex values prefaced
-    with a "#", when we are handed a string value, there is a little
-    interpretation: if the value is one of the 147 SVG named colors or
-    it starts with a "#", then it is interpreted as a value.
-
-    If a 3-tuple is provided, then it is treated as an RGB (0..255).
-    If a 4-tuple is provided, then it is treated as an RGBa (0..255), with
-    alpha as a float between 0 and 1.  (This follows the HTML5 Canvas API.)
+    * If a color is provided as a string, Bokeh determins whether this string
+      represents one of the named CSS colors or a hex value (prefaced with a
+      "#").
```

Comment:
It seems to me like Bokeh now also supports CSS4-like color strings such as `"rgba(255, 0, 127, 0.6)"` or `"rgb(0 127 0 / 1.0)"` (see example in line 105 below). Would it therefore make sense to expand this sentence:

> If a color is provided as a string, Bokeh determins whether this string represents one of the named CSS colors (such as `"red"`), a CSS4 color string (`"rgb(0,200,0)"`) or a hex value (`"#00FF00"`).